content/guides/destructuring.adoc

```diff
@@ -117,7 +128,7 @@ Say you want to print the first element on one line and the remainder on another
 ;= Amber Aaron Nick Earl Joe
 ----
 
-This binding works but even using destructuring it's pretty clunky. Instead we can use `&` to combine the tail elements into a sequence.
+This binding works but even using destructuring it's pretty clunky. Instead we can use `&` to combine the tail elements into a `seq`.
```

Comment:
`seq` is a function, "sequence" is a concept. Was right as is, imo.

Reply:
Ah, I didn't mean the seq function, it shouldn't in teletype font. I'm using this from "The Joy of Clojure, 2nd edition, p. 86": 

A _sequence_ is a sequential collection that represents a series of values that may or may not exist yet. They may be values from a concrete collection or values that are computed as necessary. A sequence may also be empty. Clojure has a simple API called _seq_ for navigating collections. It consist of two functions: first and rest.

Hence, "seq". Because what you get is evidently a sequence, But it's also a "seq".

Reply:
We use seq and sequence interchangeably in most places. Was fine as is.